src/styles/index.scss

```diff
@@ -1,3 +1,81 @@
+$calendar-padding: 10px;
+$element-day-size: 100px;
+$space-between-days: 1px;
+$max-width-calendar: (
+  7 * $element-day-size + 6 * $space-between-days + 2 * $calendar-padding
+);
+$days: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+);
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+  font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif;
```

Comment:
Ensure that the font fallback list ends with a generic family name to improve cross-browser compatibility. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), there should be a generic font family at the end of the font-family list.